stations.csv

```diff
@@ -16894,7 +16894,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 18834;Silandro/Schlanders;silandro-schlanders;8302205;;46.6259230000;10.7664880000;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;8300483;t;;f;;f;;f;;f;;f;8302205;t;S02205;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 18835;Tirano;tirano;8306719;;46.2093000000;10.1666820000;18820;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;8302436;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 18836;Venezia Mestre (Viale Stazione);venezia-mestre-viale-stazione;;;45.4821090000;12.2346420000;;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;8324253;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
-18837;Malles / Mals;malles-mals;8362035;;46.6866100000;10.5421530000;;IT;Europe/Rome;f;f;f;t;f;f;;;f;;f;8362035;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
+18837;Malles / Mals;malles-mals;8362035;;46.6866100000;10.5421530000;;IT;Europe/Rome;f;f;f;f;f;f;;;f;;f;8362035;t;;f;;f;;f;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
```

Comment:
Maybe set trenitalia_enabled to false as well?

Reply:
Uhm, why exactly?